js/kucoin.js

```diff
@@ -631,8 +633,10 @@ module.exports = class kucoin extends Exchange {
                 'strike': undefined,
                 'optionType': undefined,
                 'precision': {
-                    'amount': this.safeNumber (market, 'baseIncrement'),
-                    'price': this.safeNumber (market, 'priceIncrement'),
+                    'amount': baseIncrement,
+                    'price': priceIncrement,
+                    'base': baseIncrement,
```

Comment:
@ttodua do we need to have `base`? it's the same as `amount`

Reply:
as demonstrated [here](https://github.com/ccxt/ccxt/pull/14481) in `setCurrencyPrecisionsFromMarkets` method, the `base` & `quote` keys are needed to define currency max precisions, when  currency info/precisions are not available (i.e. not having fetchCurrencies, like binance public api or other exchange). there are different exchages - some have separate fields for `base/quote` precisions, and some don't have.

in that mentioned binance PR , I am proposing to derive a "backup of currency precisions from fetchMarkets", where through itteration i am using values of `market->precision->base/quote`. However, after your comment, I am now thinking about the following:
- if there is dedicated fields (base & quote precisions) available in api response (from fetchMarkets)  like binance has, then we set those fields. 
- if those dedicated fields are not available, should we not set the `precision ->base & quote` keys under ` and should we just use `amount & price` precisions in `setCurrencyPrecisionsFromMarkets`? if so, then kucoin doesn't have a dedicated `base & quote` precision fields available in response, and thus, i'll remove it from this PR.